config.json

```diff
@@ -23,8 +23,10 @@
       "title": "Jahresstatistik - weiteren Statistiken"
     }
   ],
-  // String or array of data provider are supported
-  "dataPath": "https://regensburg.freifunk.net/data/",
+  // Array of data provider are supported
+  "dataPath": [
+    "https://regensburg.freifunk.net/data/"
```

Comment:
could we use `./` or `./data` in future and develop branch?
you still have ffrgb-config branch

Reply:
no - the idea behind this is to load git and build and it works.
We have a origin policy (json works) and also fixedCenter, grafana is on Regensburg.

Reply:
okay